### PR TITLE
Add loved and album_loved attributes

### PR DESCRIPTION
**Pull request by shacker**

### CODE REVIEW

File: pyItunes/Library.py [hunk 1]
```diff
@@ -110,6 +110,10 @@ def getSongs(self):
 				s.has_video = attributes.get('Has Video')
 			if attributes.get('Grouping'):
 				s.grouping = attributes.get('Grouping')
+
+			s.loved = True if attributes.get('Loved') else False
+			s.album_loved = True if attributes.get('Album Loved') else False
```
Comment (shacker) [inline pyItunes/Library.py]: Note I'm doing these as one-liners, and providing a default value. Later I'll do a PR that provides default values for all of the attributes (not providing defaults forces people integrating this dependency to do tons of if/else checks).

**Comment by shacker** [comment]
The XML may or may not provide values for some fields. In the 2nd commit, I've made sure all fields return a value. This makes it possible for packages that include this as a dependency to avoid a ton of if/else processing.

`.get()` on a dictionary returns None by default, so those are easy.

When it was casting to an int or time, the statements are more verbose, since None can't be cast to int or time.

All Track fields are now one-liners, removing a lot of verbosity from Library.py

There were a ton of PEP-8 errors in the file, because it had used tabs rather than spaces (see attached screen with linter enabled). I've converted to spaces and fixed a bunch of additional PEP-8 problems (mostly missing spaces).

**Comment by shacker** [comment]
True/False attributes should be as simple as:

`s.loved = 'Loved' in attributes`

**Comment by shacker** [comment]
The only linter errors left in the file now should be line length warnings - I didn't worry about those.

**Comment by liamks** [comment]
Is this ready to merge?

**Comment by shacker** [comment]
Actually just realized I need to add the two new fields to the docs - I'll do that now. Seems to be working well for me, but I have not carefully tested every single field. Do you have any sort of test case you can run it against?

**Comment by liamks** [comment]
It looks like you changed a lot more than just loved & loved_album. Ideally the formatting would have been in a different PR - especially since there aren't tests! I'm a fan of spaces though, so I'll let it slide 👍 . Thank you or the effort!

**Comment by shacker** [comment]
I apologize for that - I knew they should have been separate PRs, but was impatient since my work depended on the first fix, so I piggy-backed the other changes. I'd like to make the same tabs --> spaces change in the other modules in this package - so you'd be open to that?

**Comment by liamks** [comment]
Yah for sure - no need to apologize! This wouldn't be an issue if I had put tests in this repo 😭!

**Comment by shacker** [comment]
PR is in. Tests would be nice, but the issue here was really about the importance of always using a python linter in your editor or IDE, so that things like tabs instead of spaces or missing spaces after commas and comments are visible and caught early.

**Pull request merged**